mastodon_aurora.css

```diff
@@ -262,6 +262,15 @@ body.lighter {
     background: #eff3f5;
 }
 
+.theme-mastoden-light .conversation--unread p {
```

Comment:
This selector probably not needed - should work without it.

Reply:
Yeah I am not seeing that in my current light theme